src/litdata/streaming/reader.py

```diff
@@ -138,6 +138,10 @@ def _apply_delete(self, chunk_index: int) -> None:
                 print(f"Skip delete {chunk_filepath} by {self._rank or 0}, current lock count: {remaining_locks}")
             return
 
+        if _DEBUG:
```

Comment:
Let's not do that. I think we need to build a better debug mode with timestamp event and a log parser to make sense of it.

Reply:
Ok, let's keep it for now until @deependujha has the new visualization wired.